dev/com.ibm.websphere.appserver.features/visibility/public/jaxrs-2.0/com.ibm.websphere.appserver.jaxrs-2.0.feature

```diff
@@ -9,5 +9,6 @@ Subsystem-Name: Java RESTful Services 2.0
 -features=com.ibm.websphere.appserver.jaxrsClient-2.0, \
  com.ibm.websphere.appserver.javaeeCompatible-7.0, \
  com.ibm.websphere.appserver.internal.jaxrs-2.0
+-bundles=com.ibm.ws.cxf.client
```

Comment:
This change shouldn't be necessary, since the `com.ibm.ws.cxf.client` bundle in included in the `jaxrsClient-2.0` feature - and that feature is included by this one.

Reply:
Ah, I meant to change that but forgot about it! Thanks for catching it!